### PR TITLE
bug fixes (empty state for agenda, bookmark icon for lunch)

### PR DESCRIPTION
**Pull request by kenyee**
- add empty state view for my agenda
- show bookmark icon for lunch
- upgrade to release version of FlexibleAdapter for empty state support

screen shot 2018-03-12 at 10 29 31 pm
screen shot 2018-03-12 at 10 23 54 pm

### CODE REVIEW

File: Droidcon-Boston/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -42,4 +42,5 @@
     <string name="str_agenda_detail_time">%1$s - %2$s</string>
     <string name="close">Close</string>
     <string name="droidconbos_twitter_handle">droidconbos</string>
+    <string name="no_bookmarked_items">No talks added to Agenda\nfor this day</string>
```
Comment (nisrulz) [inline Droidcon-Boston/app/src/main/res/values/strings.xml]: should be "No talks added to My Schedule\nfor this day", as per one of the bugs filed in trello.

File: Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/ScheduleAdapterItem.kt [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
 
 import android.support.v4.content.ContextCompat
+import android.support.v7.widget.RecyclerView
 import android.util.Log
 import android.util.TypedValue
 import android.view.View
 import android.widget.ImageView
 import android.widget.TextView
 import com.bumptech.glide.Glide
+import com.bumptech.glide.load.engine.DiskCacheStrategy
```
Comment (nisrulz) [inline Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/ScheduleAdapterItem.kt]: unused import?

File: Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/detail/AgendaDetailFragment.kt [hunk 3]
```diff
@@ -12,6 +12,7 @@ import android.view.ViewGroup
 import android.widget.ImageView
 import android.widget.RelativeLayout
 import com.bumptech.glide.Glide
+import com.bumptech.glide.load.engine.DiskCacheStrategy
```
Comment (nisrulz) [inline Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/detail/AgendaDetailFragment.kt]: unused import?

File: Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/ScheduleAdapterItem.kt (hunk 2 above)
Comment (kenyee) [reply Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/ScheduleAdapterItem.kt]: oops..forgot about that.  Was trying to see if setting the caching to All would let Glide display images on the agenda detail screen.  It supposedly downloads the full size image so it should just need to resize it but doesn't... :-P

**Pull request merged**